src/Styleguide/Components/Footer.tsx

```diff
@@ -124,19 +129,27 @@ const FooterContainer: React.SFC<FlexDirectionProps & Props> = props => {
                 </Link>
               </Serif>
             </Flex>
-            {xs && (
+            <breakpoints.xs>
               <Flex mb={1}>
                 <PolicyLinks />
               </Flex>
-            )}
+            </breakpoints.xs>
           </Flex>
           <Separator mt={1} mb={2} />
           <Flex justifyContent="space-between">
             <Flex alignItems="center" mb={4}>
-              {!xs && <Mark width="30px" height="30px" mr={2} />}
-              {xs && <Mark width="20px" height="20px" mr={2} />}
-              <Spacer mr={1} />
-              {!xs && <PolicyLinks />}
+              {/* NOTE: Responsive2 Is it possible to render out multiple instances
+                of a given breakpoint component? Like, interweve stuff? Noticing because
+                <breakpoint.xs> is used above, this xs / else isn't rendering properly
+            */}
```

Comment:
I guess this could be simplified by not having the smart `else` behavior, but rather something like `<breakpoints.not breakpoints={[breakpoints.xs, breakpoints.md]} />`.

This definitely has a lot of `breakpoints` in it, so maybe the components should be capitalized after all so that we don’t need to prefix them with `breakpoints.`